### PR TITLE
Feature/update requirements with hashes

### PR DESCRIPTION
**Pull request by gemmai95**
Changes the requirements to work with hashes this should help prevent accidental and malicious changes. (Checked on [pre-prod])

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by gemmai95** [comment]
[Turn on PIP hashed requirements.txt (3)](https://trello.com/c/ZBaaoggQ/246-turn-on-pip-hashed-requirementstxt-3)

File: requirements.txt [hunk 1]
```diff
@@ -1,12 +1,290 @@
...
+    --hash=sha256:4cc4c08a286ab78eba1b2463c96f7c1bfbda3be18277cd4405b427e18a3ef4d1 \
+    --hash=sha256:f19075a9147067bba236917d39e1f070048a18ce136e81668ecb0770a26c3f50 \
+    --hash=sha256:d7e1dd7b4c1424da544cc2cbf537baac9ad207b5488e0761571ae25ea243831a \
+    --hash=sha256:10e9b64ac3c8df3c0057033c3df57a9c94509223e402e8c1c2af4ffc7034a084 \
+    --hash=sha256:ab51ad9ffe91cca0c1141ceb06fc713471c216e11860a5dc751cc453908031ca \
+    --hash=sha256:8137381e217038bc4b7d113c9d0d1133d8081d39008b5a4f11bf129b1520c1b0
+gunicorn==19.7.1 \
+    --hash=sha256:75af03c99389535f218cc596c7de74df4763803f7b63eb09d77e92b3956b36c6 \
+    --hash=sha256:eee1169f0ca667be05db3351a0960765620dad53f53434262ff8901b68a1b622
+structlog==17.2.0 \
```
Comment (tej99) [inline requirements.txt]: Upgraded version, is that okay?

File: requirements.txt [hunk 2]
```diff
@@ -1,12 +1,290 @@
...
+    --hash=sha256:d9f709f94ee7245ba3bcab857bcc138d4b4db444916e7fbf96afb91df699fc7b \
+    --hash=sha256:21744c65a95e17caf583a9390739ad7c43cc5aada256e71fc59d4f219c98697a \
+    --hash=sha256:ffb73b245d445f0c89d6057828db9ada6468f5462854b146c09bd266494ce941 \
+    --hash=sha256:4cc4c08a286ab78eba1b2463c96f7c1bfbda3be18277cd4405b427e18a3ef4d1 \
+    --hash=sha256:f19075a9147067bba236917d39e1f070048a18ce136e81668ecb0770a26c3f50 \
+    --hash=sha256:d7e1dd7b4c1424da544cc2cbf537baac9ad207b5488e0761571ae25ea243831a \
+    --hash=sha256:10e9b64ac3c8df3c0057033c3df57a9c94509223e402e8c1c2af4ffc7034a084 \
+    --hash=sha256:ab51ad9ffe91cca0c1141ceb06fc713471c216e11860a5dc751cc453908031ca \
+    --hash=sha256:8137381e217038bc4b7d113c9d0d1133d8081d39008b5a4f11bf129b1520c1b0
+gunicorn==19.7.1 \
```
Comment (tej99) [inline requirements.txt]: Upgraded version, is that okay?

**Review by tej99 — DISMISSED** [review]

File: requirements.txt (hunk 2 above)
Comment (gemmai95) [reply requirements.txt]: It shouldn't have updated the version I will revert these back

Comment (gemmai95) [reply requirements.txt]: As it causes issues with artifactory and logging

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**